script/modify_needle

```diff
@@ -44,16 +44,15 @@ print help
 use strict;
 use warnings;
 
-use Mojo::JSON;    # booleans
+use Mojo::File qw(path);
+use Mojo::JSON;
```

Comment:
It doesn't look like that module is used at all.

Reply:
fixed